moped-editor/src/views/projects/projectsListView/components/saveViewModal.js

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import { useMutation } from "@apollo/client";
+import { Dialog, DialogTitle, DialogContent, IconButton } from "@mui/material";
+import CloseIcon from "@mui/icons-material/Close";
+import SaveViewForm from "./saveViewForm";
+import { useLocation } from "react-router-dom";
+import { ADD_USER_SAVED_VIEW } from "src/queries/project";
+
+const SaveViewModal = ({
+  showDialog,
+  setIsSaveViewModalOpen,
+  filters,
+  filtersLabels,
+  setIsViewSaved,
+  handleSnackbar,
+}) => {
+  const [saveView, { loading }] = useMutation(ADD_USER_SAVED_VIEW);
+
+  let { pathname, search } = useLocation();
+
+  const defaultDescription = filtersLabels
```

Comment:
I really like the default descriptions btw. It is nice to not have to stare at a blank input and think of something like "My super cool view" unless people really want to! 😄

Reply:
me too!